src/languages/stan.js

```diff
@@ -512,41 +410,74 @@ export default function(hljs) {
       hljs.HASH_COMMENT_MODE,
       BLOCK_COMMENT,
       {
-        // hack: in range constraints, lower must follow "<"
-        begin: /<\s*lower\s*=/,
-        keywords: 'lower'
+        scope: 'symbol',
+        match: /\s(pi|e|sqrt2|log2|log10)(?=\()/,
```

Comment:
What makes these special?  Looks like a function call to me, which would traditionally be a built-in or title.function...

Reply:
I'm treating them a bit differently because someone could name these from a type. So `e` by itself should not be highlighted but `e()` should. I'm open to these being put somewhere else.

Reply:
> So e by itself should not be highlighted but e() should

I don't have any problem with that... the `(` conditional look-ahead is GREAT... I was asking why is this symbol vs built-in or title.function... generally symbol is for a label or string like object such as symbol in ruby:

```rb
a = "string"
b = :symbol 
```

Reply:
made it `built_in`